.eslintrc

```diff
@@ -20,14 +20,17 @@
     "max-len": [
       "warn",
       {
-        "code": 150,
+        "code": 350,
         "tabWidth": 2,
         "ignoreComments": true
       }
     ],
-    "react/prop-types": "error",
+    "react/prop-types": "off",
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "no-underscore-dangle": "off",
+    "no-console": "off",
```

Comment:
I don't think we want to take this rule out, it will help to enforce not writing to console on any code that we check in. when we have a reason to actually write to console, we should be disabling with inline comments (/* eslint-disable-next-line no-console */)

Reply:
updated